oc-game-swift/Heroes/Healer.swift

```diff
@@ -8,4 +8,6 @@
 
 import Foundation
 
-
+// Healer class
+class Healer: Hero {
```

Comment:
you could add property in these classes like `description`

Reply:
I will complete hero classes with the phase 2 : the fight.